### PR TITLE
InfluxDB templates

### LINKED ISSUE #706: Document how to use InfluxDB templates
Packages allow users to import and/or export collections of InfluxDB resources (dashboards, tasks, alerts, labels, etc.). We need to document how to use the `influx pkg` command to manage packages.

### PR DESCRIPTION
**Pull request by sanderson**
Closes #706 

InfluxDB Templates documentation.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/influxdb-templates/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Use InfluxDB templates to quickly set up a fresh instance of InfluxDB, back up your
+dashboard configuration, or share your configuration with the InfluxData community.
+
+## Template manifests
+A template is defined in a single file known as a **manifest**.
+Template manifests support the following formats:
+
+- YAML
+- JSON
+- Jsonnet
```
Comment (russorat) [inline content/v2.0/influxdb-templates/_index.md]: should we link to https://jsonnet.org/?

Comment (sanderson) [reply content/v2.0/influxdb-templates/_index.md]: Sure

File: content/v2.0/influxdb-templates/create.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+- **Organization name** or **ID**
+- **Authentication token** with read access to the organization
+- **Destination path and filename** for the template manifest.
+  The filename extension determines the template format—both **YAML** (`.yml`) and
+  **JSON** (`.json`) are supported.
+
+###### Export all resources to a template
+```sh
+# Syntax
+influx pkg export all -o <org-name> -f <file-path> -t <token>
```
Comment (russorat) [inline content/v2.0/influxdb-templates/create.md]: should `-t` token be here? we don't list other global params like `--host`

**Comment by russorat** [comment]
do you want to add anything about env refs? https://github.com/influxdata/influxdb/blob/master/pkger/testdata/env_refs.yml

File: content/v2.0/influxdb-templates/create.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/influxdb-templates/create.md]: I included in the command since it's called out specifically above. I can remove it from the example.

**Comment by sanderson** [comment]
@russorat Definitely. I didn't realize that was a thing, but I'll add it.

**Pull request merged**